src/thor/triplegbuilder.cc

```diff
@@ -30,45 +30,6 @@ using namespace valhalla::thor;
 
 namespace {
 
-template <class iter>
```

Comment:
this shape trimming code has the possibility for it to only add 1 shape point for a given edge which has downstream negative consequences. there is another shape trimming method below which doesnt not suffer this problem, so i've switched to using that